packages/edit-site/src/components/header/document-actions/index.js

```diff
@@ -41,6 +43,12 @@ export default function DocumentActions( { documentTitle } ) {
 	// Title is active when there is no secondary item, or when the secondary
 	// item is inactive.
 	const isTitleActive = ! label?.length || ! isActive;
+
+	const { page } = useSelect( ( select ) => {
+		const { getPage } = select( 'core/edit-site' );
+		return { page: getPage() };
+	} );
```

Comment:
If you're just going to get one thing from `useSelect`, you can simplify this and avoid any destructuring.
```suggestion
	const page = useSelect( ( select ) => select( 'core/edit-site' ).getPage() );
```